launch.go

```diff
@@ -562,6 +562,10 @@ func main() {
 			Value:  DefaultTimeout,
 			EnvVar: "SD_BUILD_TIMEOUT",
 		},
+		cli.BoolFlag{
+			Name:  "fetch-flag",
```

Comment:
We should use easily understandable flag name.
How about `fetch-token-only` or `only-fetch-token` ?

Reply:
It makes sense. Fix flag name to `only-fetch-token`.